From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Tue, 13 Nov 2018 14:32:53 -0800
Subject: [PATCH] Put .so in python libdir

[Update for 2019-08-01]: Rebased against 1.5.0 release; moved the import
statement to a location less likely to produce future rebase conflicts.
---
 python/setup.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/python/setup.py b/python/setup.py
index dcd84ce..5137f79 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -21,6 +21,7 @@ from __future__ import absolute_import
 import os
 import sys
 from setuptools import find_packages # This must precede distutils
+from distutils.sysconfig import get_python_lib
 
 # need to use distutils.core for correct placement of cython dll
 kwargs = {}
@@ -121,7 +122,7 @@ setup(name='mxnet',
       version=__version__,
       description=open(os.path.join(CURRENT_DIR, 'README.md')).read(),
       packages=find_packages(),
-      data_files=[('mxnet', [LIB_PATH[0]])],
+      data_files=[(os.path.join(get_python_lib(1), 'mxnet'), [LIB_PATH[0]])],
       url='https://github.com/apache/incubator-mxnet',
       ext_modules=config_cython(),
       classifiers=[
